.github/actions/scale-tests/cleanup-cluster/action.yaml

```diff
@@ -0,0 +1,19 @@
+name: 'Cleanup kops cluster'
```

Comment:
Perhaps it would make sense to move the scale-tests GH action into a dedicated repository? Similar to what we have for the [little-vm-helper-action](https://github.com/cilium/little-vm-helper)

Reply:
I can see that. I think moving to the scaffolding repository could make sense, but my only hesitation is that the scale test actions are specific to the cilium repo right now and may not be used anywhere else for a while.

Could we start by leaving them here and then move them as a follow-up?

Reply:
> I can see that. I think moving to the scaffolding repository could make sense, but my only hesitation is that the scale test actions are specific to the cilium repo right now and may not be used anywhere else for a while.

Don't worry about it. We have other repositories that are only specific to the cilium repo and do not belong to this repository. I've created https://github.com/cilium/scale-tests-action for this purpose.

Reply:
Done